server/models/Request.js

```diff
@@ -0,0 +1,42 @@
+import mongoose from ("mongoose");
+import dbConnection from "./../db/mongoose";
+
+const RequestSchema = mongoose.Schema(
+    {
+        userId: [
+            {
+                type: mongoose.Schema.Types.ObjectId,
+                ref: 'User',
+                required: true
+            }
+        ],
+        requestedUserId: [
+            {
+                type: mongoose.Schema.Types.ObjectId,
+                ref: 'User',
+                required: true
+            }
+        ],
+        startTime: {
+            type: Date,
+            required: true
+        },
+        endTime: {
```

Comment:
For "startTime" and "endTIme" we can also have them as "startDate" and "endDate" since it's for the whole day, i believe, but not an issue